readme-vars.yml

```diff
@@ -51,7 +51,18 @@ opt_param_devices:
 opt_param_usage_include_ports: true
 opt_param_ports:
   - { external_port: "8920", internal_port: "8920", port_desc: "Https webUI (you need to set up your own certificate)." }
+optional_parameters: |
```

Comment:
This section probably belongs in the `app_setup_block` even though technically you are describing parameters, in this container there are other optional things like Nvidia support etc which follow a similar logic and are in that other section.